src/ansible_navigator/actions/_actions.py

```diff
@@ -52,54 +59,95 @@ def _import_all(package: str) -> None:
 
 
 def register(cls: Any) -> Any:
-    """Decorator for registering a new action"""
+    """Register an action, used as a decorator.
+
+    :param cls: The class to register
+    :returns: The class after registration
+    """
     package, _, action = cls.__module__.rpartition(".")
     pkg_info = _ACTIONS.setdefault(package, {})
     pkg_info[action] = ActionT(name=action, cls=cls, kegex=re.compile(cls.KEGEX))
     return cls
 
 
 def get(package: str, action: str) -> Callable:
-    """Get a given action"""
+    """Import and return a given action.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :returns: The action's registered class
+    """
     _import(package, action)
     return _ACTIONS[package][action].cls
 
 
 def get_factory(package: str) -> Callable:
-    """Create a get() function for one package"""
+    """Create a ``get`` function for one package.
+
+    :param package: The name of the package
+    :returns: The action's registered class
+    """
     return functools.partial(get, package)
 
 
 def kegex(package: str, action: str) -> Tuple:
-    """Return a tuple of name, kegex for a action"""
+    """Return a tuple of name, class, kegex for a action.
```

Comment:
```suggestion
    """Return a tuple of name, class, ``kegex`` for an action.
```
Maybe this'd be a bit better? Since kegex is not a word.